docs/resources/User.md

```diff
@@ -179,11 +179,14 @@ Create a new DM channel with a user. Returns a [DM channel](#DOCS_RESOURCES_CHAN
 
 ## Create Group DM % POST /users/@me/channels
 
-Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK. DMs created with this endpoint will not be shown in the Discord client
+Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK.
```

Comment:
Should we be documenting deprecated stuff at all? Deprecation usually implies we don't want the user to use them because they might go away eventually, or are not fit for purpose.

Is there a valid use case for this, or is it a feature destined to be purged randomly?

Reply:
Night has made a statement regarding it previously: https://github.com/discord/discord-api-docs/issues/1640#issuecomment-634344176